kafka: update the kafka to 2.3.0 (#50)
kudobuilder/frameworks: repository/kafka/docs/latest/install.md

> # Installing KUDO Kafka operator

Reviewer: ```suggestion
# Installing the KUDO Kafka Operator
```

Requirements:

> - [Install KUDO controller](https://kudo.dev/docs/getting-started/)

Reviewer: ```suggestion
- Install the [KUDO controller](https://kudo.dev/docs/getting-started/)
```

> - [Install KUDO cli](https://kudo.dev/docs/cli/)

Reviewer: ```suggestion
- Install the [KUDO CLI](https://kudo.dev/docs/cli/)
```

## Installing the Operator

> #### Install the Zookeeper 

Reviewer: ```suggestion
#### Install Zookeeper 
```

```
kubectl kudo install zookeeper --instance=zk
```

> #### Install the Kafka 

Reviewer: ```suggestion
#### Install Kafka 
```

```
kubectl kudo install kafka --instance=kafka
```

Verify the if the deploy plan for `--instance=kafka` is complete.
```
kubectl kudo plan status --instance=kafka
Plan(s) for "kafka" in namespace "default":
.
└── kafka (Operator-Version: "kafka-0.1.1" Active-Plan: "kafka-deploy-177524647")
    └── Plan deploy (serial strategy) [COMPLETE]
        └── Phase deploy-kafka (serial strategy) [COMPLETE]
            └── Step deploy (COMPLETE)
```

> You can check all [configuration options](./configuration.md)

Reviewer: ```suggestion
You can view all configuration options [here](./configuration.md)
```
